### ISSUE TITLE
Add Hydrus version in various places for troubleshooting

### ISSUE THREAD

**Issue by Zweibach** [hydrusnetwork/hydrus#447]
Add Hydrus version to as many places as possible in the various logs and windows and titlebars to more easily allow helpers identify people on old Hydrus versions. Make all error reports and messages include Hydrus version, including tracebacks if possible.

A number of times we've encountered people with really weird issues only to discover they're on a version up to years out of date and the problem was long since fixed. If we could at a glance tell that they are on old versions this could reduce headache a fair deal.

**Comment by floogulinc** [comment]
I don't think it would be hard for Hydrus to optionally check the GitHub API for new releases and notify the user so that might be an option.
